tests_e2e/orchestrator/runbook.yml

```diff
@@ -33,6 +48,25 @@ variable:
   - name: default_location
     value: "westus2"
 
+  #
+  # These variables define parameters for the AgentTestSuite; see the test wiki for details
+  #
+  # The test suites to execute
+  - name: test_suites
```

Comment:
All the parameters that we pass to lisa as variable should be define here even though we don't use them in this file?

Reply:
yes